pandas/tests/test_strings.py

```diff
@@ -1208,6 +1209,21 @@ def test_match(self):
         exp = Series([True, np.nan, np.nan])
         tm.assert_series_equal(exp, res)
 
+    def test_fullmatch(self):
+        values = Series(["fooBAD__barBAD", "BAD_BADleroybrown", np.nan, "foo"])
```

Comment:
can you add the issue number here as a comment

Reply:
Added a comment with the issue number in commit 2f94e0c .
The syntax I used was:
```
# GH 32806
```
Please let me know if I should be using a different format to avoid messing up any automated tooling for these issue references.